planetiler-core/src/main/java/com/onthegomap/planetiler/archive/TileArchiveMetadata.java

```diff
@@ -93,7 +96,7 @@ public TileArchiveMetadata(Profile profile, PlanetilerConfig config, List<LayerA
       config.minzoom(),
       config.maxzoom(),
       vectorLayers == null ? null : new TileArchiveMetadataJson(vectorLayers),
-      mapWithBuildInfo(),
+      mergeMaps(mapWithBuildInfo(),profile.extraArchiveMetadata()),
```

Comment:
I think I'd expect the profile to provide defaults, then args to layer on top of that, so let's either reverse this or the behavior of mergeMaps?

Reply:
shouldn't `mapWithBuildInfo` just be the `planetiler:*` metadata? If I define those in my profile (or command line args passed into my profile), they ought to override the defaults, I don't know why someone would do that though.

Reply:
Oh right! Yeah this looks good then. The command-line args get copied in just above this.